### ISSUE TITLE
Landing page

### ISSUE THREAD

**Issue by wayferer** [borna761/ClusterMaps#35]

**Comment by borna761** [comment]
Need some ideas for what to do here... maybe leave the Test cluster as available so that people can see what it does as a start?

**Comment by wayferer** [comment]
So basically what is shown on the home page once logged in but using the test data?

**Comment by borna761** [comment]
yeah, and not allow the user to change the cluster. and then later on I'll try to figure out what kind of text to put and where.

**Comment by wayferer** [comment]
@borna761 Hey. I've add a home page on the `home` branch and fixed up the routes and things but I'm not sure how we should go about setting up the example data. Thoughts?
